sentry/sentry-symfony/4.6/config/packages/sentry.yaml

```
@@ -1,6 +1,11 @@
 when@prod:
     sentry:
         dsn: '%env(SENTRY_DSN)%'
+        # this hooks into critical paths of the framework (and vendors) to perform
+        # automatic instrumentation (there might be some performance penalty)
+        # https://docs.sentry.io/platforms/php/guides/symfony/performance/instrumentation/automatic-instrumentation/
+        tracing:
+            enabled: true
```

I would set it to false by default as it impacts performance (and we've been hit by that significantly in the past for some of our projects).

```suggestion
            enabled: false
```

---

OK, I've just set it to `false` by default.